### ISSUE TITLE
Option to generate enums with ALL_CAPS members

### ISSUE THREAD

**Issue by tfatkuli** [koxudaxi/datamodel-code-generator#783]
**Is your feature request related to a problem? Please describe.**
Currently, the generated enums look like this:
```python
class Status(Enum):
    first_status = "status1"
    second_status = "status2"
```
**Describe the solution you'd like**
Would be nice to have an option like `--capitalise-enum-members` to produce result like this:
```python
class Status(Enum):
    FIRST_STATUS = "status1"
    SECOND_STATUS = "status2"
```

**Comment by koxudaxi** [comment]
@tfatkuli 
I'm sorry for my too-late reply.
I have released the new version as 0.14.1.
The version has this option.
```
  --capitalise-enum-members
                        Capitalize field names on enum
```

**Comment by tfatkuli** [comment]
@koxudaxi
Cool! Thanks for finally making it possible!
